components/Banner/Banner.tsx

```diff
@@ -1,26 +1,33 @@
 /* eslint-disable camelcase */
-import { Icon } from "@equinor/eds-core-react"
+import { Icon, Typography } from "@equinor/eds-core-react"
 import { warning_filled } from "@equinor/eds-icons"
+import { tokens } from "@equinor/eds-tokens"
 import { FunctionComponent, ReactNode } from "react"
+import styled from "styled-components"
 
 type Props = {
   variant: string
   children: ReactNode
 }
 
-export const Banner: FunctionComponent<Props> = ({ variant, children }) => {
-  const classNames = [
-    "banner",
-    variant === "danger" ? "banner--danger" : "banner--warning",
-  ]
-    .join(" ")
+const BannerContainer = styled.div<{ variant: string }>`
+    background-color: ${({ variant }) => (variant === "danger" ? tokens.colors.ui.background__danger.rgba : tokens.colors.ui.background__warning.rgba)};
+    display: flex;
+    justify-content: start;
+    gap: 15px;
+`
 
-  return (
-    <div className={classNames}>
-      <div className="banner_icon">
-        <Icon color={variant === "danger" ? "#cc0000" : "#AD6200"} data={warning_filled} />
-      </div>
-      <div className="banner_container">{children}</div>
-    </div>
-  )
-}
+const BannerIcon = styled.div`
+    padding-left: 5px;
+`
+
+export const Banner: FunctionComponent<Props> = ({ variant, children }) => (
+  <BannerContainer variant={variant}>
```

Comment:
The banner container still needs some padding, and that one was removed by an accident